### PR TITLE
Removing aditional files

### PR DESCRIPTION
**Pull request by juanjoaquino05**
Leaving all logic in Mastermind.cs file

### CODE REVIEW

**Comment by granalberto** [comment]
`The type or namespace name 'Exception' could not be found (are you missing a using directive or an assembly reference?)`

The message is clear.

**Comment by juanjoaquino05** [comment]
You can try it now.

File: CSharpMatermindGameChallenge/Mastermind.cs [hunk 1]
```diff
@@ -2,7 +2,8 @@
 namespace CSharpMatermindGameChallenge
```
Comment (granalberto) [inline CSharpMatermindGameChallenge/Mastermind.cs]: Please change the namespace to `Mastermind`

**Review by granalberto — CHANGES_REQUESTED** [review]

**Comment by juanjoaquino05** [comment]
Done

**Comment by granalberto** [comment]
Now my tests are running. Let's close this one.

**Pull request merged**